src/node/node_state.h

```diff
@@ -130,7 +130,7 @@ namespace ccf
     kv::Version recovery_v;
     crypto::Sha256Hash recovery_root;
     std::vector<kv::Version> view_history;
-    consensus::Index last_recovered_signed_idx = 1;
+    consensus::Index last_recovered_signed_idx = 0;
```

Comment:
This change looks suspicious to me, because `last_recovered_signed_idx` isn't mentioned anywhere else in this changeset, but it's probably fine :-)

Reply:
Yes, I changed this in the abandoned PR #3525 as I used `last_recovered_signed_idx` but not in this PR. However, `last_recovered_signed_idx` should be initialised at `0` and not `1` and this shouldn't have any impact on the recovery procedure.